phpBB/assets/javascript/installer.js

```diff
@@ -357,10 +364,21 @@
 			$('#loading_indicator').css('display', 'none');
 			resetPolling();
 
+			var timeoutDetected = !transmissionOver;
+
 			if (refreshRequested) {
 				refreshRequested = false;
 				doRefresh();
 			}
+
+			if (timeoutDetected) {
+				addMessage('error',
+					[{
+						title:'The installer detected a timeout.',
+						description: 'The installer has detected a timeout, you may try to refresh the page, that may lead to data corruption. We suggest that you either increase your timeout settings or try to use the CLI.'
```

Comment:
Why do you think that doesn't matter?
Also, pass the lang vars to the installer.html-whatever-template in some javascript code and then access them from the installer.js? That's how I'd approach this.